third_party/versions.bzl

```diff
@@ -153,16 +153,16 @@ HTTP_DEPENDENCY_VERSIONS = {
         "version": "4.5",
     },
     "rules_java": {
-        "sha": "34b41ec683e67253043ab1a3d1e8b7c61e4e8edefbcad485381328c934d072fe",
-        "version": "4.0.0",
+        "sha": "c73336802d0b4882e40770666ad055212df4ea62cfa6edf9cb0f9d29828a0934",
+        "version": "5.3.5",
     },
     "rules_jvm": {
         "sha": "c4cd0fd413b43785494b986fdfeec5bb47eddca196af5a2a98061faab83ed7b2",
         "version": "5.1",
     },
-    "rules_kotlin": {
-        "sha": "12d22a3d9cbcf00f2e2d8f0683ba87d3823cb8c7f6837568dd7e48846e023307",
-        "version": "v1.5.0",
+    "rules_kotlin_prerelease": {
+        "sha": "1d872b9c6546f0f737a356d873b164d70282760fe4c880349770abc9e494c9ce",
+        "version": "v1.7.2",
```

Comment:
Document version upgrade & removal of patch.

This is a special fork (link to fork & explain why). Explain how this will change in future PRs.

Reply:
Done.